dbschema/default.esdl

```diff
@@ -1,49 +1,99 @@
 module default {
-  global currentGroupID: str;
-  global currentGroup := (
+  global current_group_id: str;
```

Comment:
I know it's more EdgeQL-y to use snake_case, but for JS/TS projects, we've been leaning toward matching the common idiom there and using camelCase.

Reply:
ahhhh why have I done this to myself? I've been finding snake_case easier to read, but you're right, the ecosystem points to camelCase

Should I go for it?